modules/health_quest/config/routes.rb

```diff
@@ -4,5 +4,9 @@
   namespace :v0, defaults: { format: :json } do
     resources :appointments, only: %i[index show] do
     end
+    resources :pgd_questionnaires, only: %i[show] do
```

Comment:
no block is necessary here, this could be just `resources :pgd_questionnaires, only: %i[show]`

Reply:
ok, I have addressed the requested changes